platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/jasperdb/VirtualDataSourceJasperDB.java

```diff
@@ -350,42 +348,26 @@ public VirtualDataSourceJasperDB(
         this.label = label;
         // updated count of open databases
         COUNT_OF_OPEN_DATABASES.increment();
-        // create thread group with label
-        final ThreadGroup threadGroup = new ThreadGroup("JasperDB-" + label);
         // create scheduledThreadPool for executing merges
-        mergingExecutor = new ScheduledThreadPoolExecutor(
-                NUMBER_OF_MERGING_THREADS,
-                new ThreadConfiguration(getStaticThreadManager())
-                        .setThreadGroup(threadGroup)
-                        .setComponent(JASPER_DB_COMPONENT)
-                        .setThreadName("Merging")
-                        .setExceptionHandler((t, ex) -> logger.error(
-                                EXCEPTION.getMarker(), "[{}] Uncaught exception during merging", label, ex))
-                        .buildFactory());
+        mergingExecutor = getStaticThreadManager()
+                .createScheduledThreadPool(JASPER_DB_COMPONENT + ": Merging", NUMBER_OF_MERGING_THREADS);
```

Comment:
Exception handler is missed here. Also, it would be great to preserve `label` here. Previously it was in the thread group name (I don't really think we needed a thread group per data source), now it's nowhere. Maybe put it next to `JASPER_DB_COMPONENT`, something like this:
```
.createScheduledThreadPool(JASPER_DB_COMPONENT + "/" + label + ": Merging", NUMBER_OF_MERGING_THREADS);
```
?

Reply:
Fun fact, the standard implementation of a scheduled executor service overrides the uncaught exception handler, meaning that the one we were setting previously was not actually used. For the sake of clarity, I have removed API that allows for an uncaught exception handler to be set in situations where it will be ignored.

I will do another pass and make sure I fix the names to be consistent with the way it was done previously. Changing the name of the pool was not intentional.